core/src/components/datetime/datetime.tsx

```diff
@@ -1848,7 +1848,11 @@ export class Datetime implements ComponentInterface {
             <ion-item button detail={false} lines="none" onClick={() => this.toggleMonthAndYearView()}>
               <ion-label>
                 {getMonthAndYear(this.locale, this.workingParts)}{' '}
-                <ion-icon icon={this.showMonthAndYear ? expandedIcon : collapsedIcon} lazy={false}></ion-icon>
+                <ion-icon
+                  aria-hidden="true"
```

Comment:
This icon was being announced as "image" when the section was being read. Hiding it from screen readers as that information is not valuable or useful.